tests/unit/source_code/test_notebook.py

```diff
@@ -228,30 +228,30 @@ def test_notebook_builds_python_dependency_graph(mock_path_lookup) -> None:
 
```

Comment:
These tests required updating as the known dependencies are now registered where before they were not